{"name": "Retrieve pet data", "status": "broken", "statusDetails": {"message": "json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n", "trace": "  File \"d:\\programs\\python3\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"d:\\programs\\python3\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\Steps.py\", line 29, in step_impl\n    Functions.print_api_response(self)\n  File \"D:\\udemy.python.ApiRestFramework\\src\\functions\\Functions.py\", line 56, in print_api_response\n    self.json_response = json.loads(self._response.text)\n  File \"d:\\programs\\python3\\lib\\json\\__init__.py\", line 348, in loads\n    return _default_decoder.decode(s)\n  File \"d:\\programs\\python3\\lib\\json\\decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"d:\\programs\\python3\\lib\\json\\decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\n"}, "steps": [{"name": "Given I connect with endpoint pet/10", "status": "passed", "start": 1583561821943, "stop": 1583561821945}, {"name": "When I do a Get", "status": "passed", "start": 1583561821945, "stop": 1583561822294}, {"name": "Then I print out the results of the response", "status": "broken", "statusDetails": {"message": "json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n", "trace": "  File \"d:\\programs\\python3\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"d:\\programs\\python3\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\Steps.py\", line 29, in step_impl\n    Functions.print_api_response(self)\n  File \"D:\\udemy.python.ApiRestFramework\\src\\functions\\Functions.py\", line 56, in print_api_response\n    self.json_response = json.loads(self._response.text)\n  File \"d:\\programs\\python3\\lib\\json\\__init__.py\", line 348, in loads\n    return _default_decoder.decode(s)\n  File \"d:\\programs\\python3\\lib\\json\\decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"d:\\programs\\python3\\lib\\json\\decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\n"}, "start": 1583561822294, "stop": 1583561822298}, {"name": "Then The api response 404 Dont Exist", "status": "skipped", "start": 1583561822302, "stop": 1583561822302}], "start": 1583561821933, "stop": 1583561822302, "uuid": "ac6e11b9-d0db-4190-aa55-3ef53729ba77", "historyId": "633856d690ddcd939d4f770d9d9b11b7", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "Integration"}, {"name": "tag", "value": "WebApi"}, {"name": "feature", "value": "Test WebApi"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}